autopush_rs/src/util/ddb_helpers.rs

```diff
@@ -869,6 +885,27 @@ impl DynamoStorage {
         Box::new(response)
     }
 
+    pub fn unregister(
+        &self,
+        uaid: &Uuid,
+        channel_id: &Uuid,
+        message_month: &str,
+        code: u32,
+    ) -> MyFuture<bool> {
+        let ddb = self.ddb.clone();
+        let response = DynamoStorage::unregister_channel_id(ddb, uaid, channel_id, message_month)
+            .and_then(move |_| -> MyFuture<_> { Box::new(future::ok(true)) })
+            .or_else(move |_| -> MyFuture<_> { Box::new(future::ok(false)) });
+        if code != 200 {
+            info!("Unregister";
```

Comment:
did we want the info! for register then? You said we didn't need it in the other PR

Reply:
I don't think we care about successful registers (they all are), but unsubscribes are different cause they can happen for non-user-driven reasons like quota expiration, etc. The Python code always logged the unregisters, which like the registers is a bit pointless. But we definitely want to know about the codes involved in non-user-driven unregisters. Though, I think we want a tagged metric now that you mention it, not a log statement.